vital/algo/algorithm.cxx

```diff
@@ -232,10 +232,21 @@ ::check_nested_algo_configuration( std::string const& type_name,
       std::string reg_name;
       if ( a_fact->get_attribute( kwiver::vital::plugin_factory::PLUGIN_NAME, reg_name ) )
       {
+        if (first)
+        {
+          first = false;
+          msg << "   Available implementations are:";
+        }
+
         msg << "\n      " << reg_name;
       }
     }
 
+    if (first)
+    {
+      msg << "   There are no alternate implementations available.";
```

Comment:
I'm not sure that "alternate" makes sense here.  May use "   There are no algorithm implementations available."

Reply:
Done